### PR TITLE
Update README.md - remove content and refer to developer.readme.com, update platform support

### PR DESCRIPTION
**Pull request by bitbalint**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,48 +17,23 @@ Official React Native bindings for Bitmovin's mobile Player SDKs.
...
 This library requires at least React Native 0.64+ and React 17+ to work properly. The currently supported platforms are:
 
-- iOS 12.0+
-- tvOS 12.0+
-- Android API 16+
-- Android TV API 17+
-- Fire TV (just make sure the Android API level is at least 17+)
+- iOS 14.0+
+- tvOS 14.0+
+- Android API Level 16+ (note that DRM-protected streams require API Level 19+)
```
Comment (rolandkakonyi) [inline README.md:32]: Based on the project settings we require min SDK version 21 on Android

see: https://github.com/bitmovin/bitmovin-player-react-native/blob/development/android/build.gradle#LL33C1-L34C1

@zigavehovec to confirm

Comment (zigavehovec) [reply README.md:32]: Confirming. React native version `0.64` changed minSdkVersion version to 21 so it was not possible to compile with minSdkVersion set at 16.

Changelog: https://reactnative.dev/blog/2021/03/12/version-0.64#major-dependency-version-changes

Comment (bitbalint) [reply README.md:32]: Thanks, this also applies to Android TV? @zigavehovec

**Review by rolandkakonyi — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (zigavehovec) [reply README.md]: Yes, this also applies to Android TV.

**Review by zigavehovec — APPROVED** [review]
👍

**Pull request merged**